components/src/web-components/visualization/gs-wastewater-mutations-over-time.stories.ts

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './gs-wastewater-mutations-over-time';
+import '../app';
+import { withComponentDocs } from '../../../.storybook/ComponentDocsBlock';
+import { WISE_DETAILS_ENDPOINT, WISE_LAPIS_URL } from '../../constants';
+import details from '../../preact/wastewater/mutationsOverTime/__mockData__/details.json';
+import { type WastewaterMutationsOverTimeProps } from '../../preact/wastewater/mutationsOverTime/wastewater-mutations-over-time';
+
+const codeExample = String.raw`
+<gs-wastewater-mutations-over-time
+    lapisFilter='{ "dateFrom": "2024-01-01" }'
+    sequenceType='nucleotide'
+    width='100%'
+    height='700px'
+></gs-wastewater-mutations-over-time>`;
+
+const meta: Meta<Required<WastewaterMutationsOverTimeProps>> = {
+    title: 'Visualization/Wastewater mutations over time',
```

Comment:
```suggestion
    title: 'Wastewater/Mutations over time',
```
I would put the specific wastwater components in a different folder, so it is clear, that they are not used for "normal" people.